src/lib/metrics.ts

```diff
@@ -259,6 +265,42 @@ export default class MetricsMonitor {
                         .labels({ range: clientStat.range })
                         .set(clientStat.count),
                 );
+
+                rateLimits.reset();
```

Comment:
Any particular reason we need to reset it here? Will we have gotten any inputs before this point that we need to get rid of?

Reply:
I'm still not sure, I just followed the patterns for the rest of our static metrics.

Reply:
In my mind since it's a gauge there shouldn't really be a need to reset it.